content/ch-quantum-hardware/cQED-JC-SW.ipynb

```diff
@@ -258,7 +258,7 @@
    ],
    "source": [
     "# define J-C Hamiltonian in terms of diagonal and non-block diagonal terms\n",
-    "H0 = wr*Dagger(a)*a - (1/2)*wq*sz; \n",
+    "H0 = wr*Dagger(a)*a + (1/2)*wq*sz; \n",
```

Comment:
This is correct here, the error is with the original Hamiltonain.

Reply:
If the - is right, then the issue with the delta of the original post persists.
To be more precise: In the both equations there is no minus at all, so it can not imply the A and B with the definition of the given Delta (with an the minus).

Reply:
Thanks for the careful look @HTransistor! I see now that the problem is arising due to the \sigma_+ and \sigma_- being defined for spins in `sympy`, rather than qubit levels, so there is a discrepancy between the +/-. Then you are indeed correct that the A and B should have the minus signs. I am not sure what the most elegant solution for replacing the qubit raising and lowering operators is, however.

Reply:
Alright, good that we find that one. I renamed the ladder operators (inversed, and made a comment in the code) and reverted the old change in H0, in the newest PR. Now it should be fine.